hokusai/production.yml

```diff
@@ -103,3 +103,111 @@ spec:
                     operator: In
                     values:
                     - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-hokusai-run-pods
+spec:
+  schedule: "30 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          annotations:
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+        spec:
+          containers:
+            - name: opstools-cleanup-hokusai-run-pods
+              image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/opstools:production
+              command: ["src/kubernetes_cleanup_pods/cleanup.py", "6", "--name=-hokusai-run", "--force"]
```

Comment:
including an equals sign `--name=-hokusai...` to force argparse to interpret leading `-` as a value